src/ansys/fluent/core/graphics_definitions/pyvista/pyvista_objects.py

```diff
@@ -0,0 +1,276 @@
+"""Module providing visualization objects for PyVista."""
```

Comment:
In pyfluent-ui we are using `Graphics` from here while we are not using pyvista. Similarly from the other package we are using `Plot` but not matplotlib. So, in moving this, I believe that we should also move `Graphics` and `Plot` out of such namespaces, so that everything is clearer in the client code.